parts/k8s/containeraddons/node-problem-detector.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: node-problem-detector
+  namespace: kube-system
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Consider adding a [Service](https://gist.github.com/juan-lee/b4a98b9aa7384afc74e311134ee98cac) and the --prometheus-address arg to expose npd's metrics endpoint. Also, I think we also need the /dev/kmsg mount.

Reply:
I added all the above.